Vizzuality/heco-invest
# Add map layers toggle component to Discover Projects' map

## IMPORTANT

This PR includes (because it depends on) #311 and #316 .

## Description

This PR adds a selector so the user can select contextual layers to display on the Discover page map. 

## Testing instructions

Verify that:  
- The user can click on the "Map layers" button and a selector opens  
- The user can toggle layers by clicking on them  
  _Note: the layers display is not part of this task, so some may not work. They've been added for demonstration/testing purposes._  
- If the user clicks outside the selector closes  

## Tracking

[LET-654](https://vizzuality.atlassian.net/browse/LET-654)

## Screenshot  

<img width="1678" alt="Screenshot 2022-06-24 at 10 38 24" src="https://user-images.githubusercontent.com/6273795/175508718-0afadf0d-cdb7-428b-81cf-2bb0328e16a3.png">

## Video

https://user-images.githubusercontent.com/6273795/175551759-6ab5e996-7733-4125-a1d2-dd3e6783b747.mp4
